Sources/StreamChatUI/Composer/Suggestions/ChatSuggestionsVC.swift

```diff
@@ -81,12 +81,14 @@ open class ChatSuggestionsVC: _ViewController,
             options: [.new],
             changeHandler: { [weak self] collectionView, change in
                 guard let self = self, let newSize = change.newValue else { return }
+                guard !collectionView.isTrackingOrDecelerating else { return }
```

Comment:
Some customers have reported issues that while scrolling the suggestion list the suggestion view was changing height. This change aims to avoid these kinds of updates.